pkg/controller/jobframework/defaults.go

```diff
@@ -45,14 +45,13 @@ func ApplyDefaultForSuspend(ctx context.Context, job GenericJob, k8sClient clien
 	return nil
 }
 
-// +kubebuilder:rbac:groups="apps",resources=replicasets,verbs=get;list;watch
```

Comment:
Moved to pkg/controller/jobs/deployment/deployment_controller.go

Reply:
sgtm, we could even make another cleanup PR of this, but no strong view.